core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1814,4 +1815,110 @@ class PlaintextConsumerTest extends BaseConsumerTest {
         s"The current assignment is ${consumer.assignment()}")
   }
 
+  @Test
+  def testConsumingWithNullGroupId(): Unit = {
```

Comment:
Just to complete the integration picture, could we make a call to `committed` and assert the exception?